de.dashup.model/src/test/java/de/dashup/test/UnitTestUtil.java

```diff
@@ -57,11 +57,68 @@ static void setUpTestDataset(Database database) throws SQLException {
         Assertions.assertEquals(1, database.get(Database.Table.USERS_SETTINGS, new HashMap<>()).length());
 
         testDataMap.clear();
-        testDataMap.put("token","t8KJgrLLuP51Tilw6SiXjqoyM0EFX6OxrbTG5giYbXRPoJk1dUOoUHRHbx7lTPiD");
-        testDataMap.put("user_id","2");
-        testDataMap.put("expire_date","2099-01-01");
+        testDataMap.put("token", "t8KJgrLLuP51Tilw6SiXjqoyM0EFX6OxrbTG5giYbXRPoJk1dUOoUHRHbx7lTPiD");
+        testDataMap.put("user_id", "2");
+        testDataMap.put("expire_date", "2099-01-01");
         database.insert(Database.Table.USERS_TOKENS, testDataMap);
 
         Assertions.assertEquals(1, database.get(Database.Table.USERS_TOKENS, new HashMap<>()).length());
     }
+
+    static void setUpTestDashup(Database database) throws SQLException {
```

Comment:
We might consider merging this method into `UnitTestUtil#setUpTestDataset`.

Reply:
For every test a dashboard would be created, even if it is not needed